packages/ubuntu_bootstrap/lib/pages/source/source_page.dart

```diff
@@ -39,20 +39,26 @@ class SourcePage extends ConsumerWidget {
           ...model.sources
               .map((source) => Align(
                     alignment: AlignmentDirectional.centerStart,
-                    child: YaruRadioButton<String>(
-                      title: Text(source.localizeTitle(context)),
-                      subtitle: Text(source.localizeSubtitle(context)),
-                      contentPadding: kWizardPadding,
-                      value: source.id,
-                      groupValue: model.sourceId,
-                      onChanged: model.setSourceId,
+                    child: Visibility(
+                      visible: model.sources.length > 1,
```

Comment:
I bet the compiler can optimize this out, but it'd rather store this once in the beginning of the `Widget build(...` method and reuse in here and in the other `Visibility` widget, thus avoiding the repetition.

```dart
final model = ref.watch(sourceModelProvider);
final showRadioButtons = model.sources.length > 1;
```

and then

```dart
                    child: Visibility(
                      visible: showRadioButtons,
...
```

Reply:
I still need to compute the button because it is then used to retrieve the value of the source. Removing it entirely would require a more significant refactoring of the code.

Reply:
Thanks for your comment. I updated the PR accordingly.